espanol/muralla/apuntalar/abrid_fuego/abrid_fuego.md

```diff
@@ -0,0 +1 @@
+En ese momento, una lluvia de flechas cayó sobre los visitantes, creando zumbidos estremecedores que se conjugaban con los gritos de aquellos alcanzados por ellas.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.